packages/edit-site/src/components/header/index.js

```diff
@@ -88,7 +93,11 @@ export default function Header( {
 		! isLargeViewport || deviceType !== 'Desktop' || hasFixedToolbar;
 
 	return (
-		<div className="edit-site-header">
+		<div
+			className={ classnames( 'edit-site-header', {
+				'navigation-open': isNavigationOpen,
```

Comment:
should this go further up in the DOM? I feel like there is some precedent to set these at the `body` tag (e.g. `is-full-screen`)

Reply:
This is a good point. Normally I'd say wait until there is a business use case, but I think we can safely predict that other components and styles will eventually need to leverage the same `is-navigation-open` flag.

Would you mind pointing me in the right direction? I'm having a little bit of trouble finding the `body` tag. If not, I'll go looking tomorrow.

Reply:
I just meant this: https://www.w3schools.com/tags/tag_body.asp. It's already where many similar styles like this exist :) I'm not sure where those styles are set; I'd probably search for `is-navigation-open` to see where it is added

Reply:
> I just meant this: https://www.w3schools.com/tags/tag_body.asp.

Thanks Noah! Yeah, I was following the idea of the body tag. It was more of a question of where the styles are generated to add to the body tag, and I was wondering if it was anywhere obvious like an `index.html` in the Gutenberg repo.

I'll go searching now 👍

Reply:
Hmmm...it looks like one of the patterns we use to establish `is-fullscreen-mode` is here `packages/interface/src/components/fullscreen-mode/index.js:36`

```javascript
if ( isActive ) {
	document.body.classList.add( 'is-fullscreen-mode' );
} else {
	document.body.classList.remove( 'is-fullscreen-mode' );
}
```

Based on a comment in the same file on line 11, the classname is also set in PHP somewhere when loading Gutenberg. I'll spend a little bit looking for the implementation, but I think we'll end up with a similar implementation as for fullscreen mode, where we set the classname with `document.body.classList.add()`

Reply:
Yep, that makes sense to me. I do not believe that the reference to PHP is applicable in this situation :) I do know that historically, classes like that would have been added in PHP for other admin screens or even the front end. It looks like that comment is just handling compatibility with a different flag which can cause issues

Reply:
I added the body classname update in 13aad6b